### PR TITLE
Update PAT documentation

### PR DESCRIPTION
**Pull request by Americas**

### CODE REVIEW

File: _authentication.md [hunk 1]
```diff
@@ -100,6 +100,16 @@ curl https://api.uphold.com/v0/me/tokens \
   -d '{ "description": "My command line script" }'
 ```
 
+> The above command returns the following JSON:
+
+```json
+{
+    "accessToken":"c386ae9c4557c1c661b15911071b06d9e6c3fc9a",
+    "description":"token 1",
```
Comment (rplopes) [inline _authentication.md]: Wrong description, based on the example request above.

File: _authentication.md [hunk 2]
```diff
@@ -138,12 +148,12 @@ token     | yes      | The PAT you wish to revoke.
 
 ```bash
 curl https://api.uphold.com/v0/me \
-  -u 41ee8b1fa14042e031fe304bb4793b54e6576d19b306dc205136172b80d59d20:X-OAuth-Basic
+  -H "Authorization: Bearer <token>"
 ```
 
-A PAT may be used for authenticating a request via the HTTP Basic Authentication scheme.
+A PAT may be used for authenticating a request via the oAuth scheme.
```
Comment (rplopes) [inline _authentication.md]: OAuth, as used elsewhere in this document.

File: _authentication.md (hunk 1 above)
Comment (Americas) [reply _authentication.md]: Fixed.

File: _authentication.md (hunk 2 above)
Comment (Americas) [reply _authentication.md]: Fixed.

**Comment by rplopes** [comment]
Rebase?

**Comment by Americas** [comment]
@rplopes done

**Pull request merged**